Mathlib/Order/WellQuasiOrder.lean

```diff
@@ -0,0 +1,151 @@
+/-
+Copyright (c) 2025 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios, Aaron Anderson
+-/
+import Mathlib.Data.Fintype.Card
+import Mathlib.Data.Set.Finite.Basic
+import Mathlib.Order.Antichain
+import Mathlib.Order.OrderIsoNat
+
+/-!
+# Well quasi-orders
+
+A well quasi-order (WQO) is a relation such that any infinite sequence contains an infinite
+monotonic subsequence. For a preorder, this is equivalent to having a well-founded order with no
```

Comment:
```suggestion
subsequence of related elements. For a preorder, this is equivalent to having a well-founded order with no
```
since you aren't assuming transitivity in this sentence